pkg/operator/controller/dns/controller.go

```diff
@@ -640,24 +652,37 @@ func (r *reconciler) customCABundle() (string, error) {
 	return caBundle, nil
 }
 
-//getIbmDNSProvider will intialize the IBM DNS provider
-func getIbmDNSProvider(dnsConfig *configv1.DNS, creds *corev1.Secret, cisInstanceCRN, userAgent string) (*ibmdns.Provider,
-	error) {
+//getIbmDNSProvider initializes and returns an IBM DNS provider instance.
```

Comment:
Sorry for being so persnickety, but...
```suggestion
// getIbmDNSProvider initializes and returns an IBM DNS provider instance.
```

Reply:
Fixed.